requirements.txt

```diff
@@ -1,3 +1,2 @@
-docutils==0.16
-sphinx-prompt==1.1.0
-sphinx==2.4.4
+sphinx-prompt>=1.1
+sphinx>=2
```

Comment:
I appreciate this change and have been convinced that it is the right thing to do.